src/background.js

```diff
@@ -0,0 +1,5 @@
+const dummy = {
+  name: 'john doe',
+};
+
+export default dummy;
```

Comment:
To use ES6 modules in a background page we will have to use `background.page` instead of `background.scripts` in the manifest.json file and define a background HTML page and then load the scripts into it as ES6 modules [using script tags with type "module"].